src/components/Project/ProjectRow.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
+import {Link} from 'react-router-dom';
 
 export default function ProjectRow(props) {
   return (
     <tr>
       <td>{props.project.id}</td>
-      <td className='text-warning'>{props.project.title}</td>
+      <td className='text-warning'>
+        <Link to='/project-detail' className='text-warning'>
```

Comment:
Also, if possible, please add id into route.

Reply:
Postponed https://trello.com/c/BgfSPOet

Reply:
https://trello.com/c/fpnoYKTU

Reply:
```js
<Link to={`/projects/${props.project.id}`} className='text-info'>
          {props.project.title}
        </Link>
```
I passed `id` to `<Link>`